source/features/link-to-github-io.tsx

```diff
@@ -30,15 +30,19 @@ function initRepo(signal: AbortSignal): void {
 }
 
 function initRepoList(signal: AbortSignal): void {
-	observe('a[href$=".github.io"][itemprop="name codeRepository"]', addRepoListLink, {signal});
+	observe([
+		// Earlier GitHub Pages were hosted on github.com #6228
+		'a[itemprop="name codeRepository"][href$=".github.com"]',
+		'a[itemprop="name codeRepository"][href$=".github.io"]',
+	], addRepoListLink, {signal});
 }
 
 void features.add(import.meta.url, {
 	include: [
 		pageDetect.hasRepoHeader,
 	],
 	asLongAs: [
-		() => Boolean(getRepo()?.name.endsWith('.github.io')),
+		() => /\.github\.(io|com)$/.test(getRepo()?.name!),
```

Comment:
```suggestion
		() => /\.github\.(io|com)$/.test(getRepo()?.name ?? 'shush ts'),
```

Reply:
This is better than calling getRepo twice. Please use this solution

Reply:
I have applied the suggested changes. 🙂